### PR TITLE
Backspace can unselect all tags now

### PR DESCRIPTION
**Pull request by Tagirijus**
I reset my branch, since with my recent PR #480 I accidentally had all my recent changes combined in one PR, which was not my intention. Also I now added things to the GUI to properly reflect the new feature.

### CODE REVIEW

**Comment by Tagirijus** [comment]
Ok now only one test failed, the linting one. Did I do something wrong with the code formatting or so?

File: timetagger/app/front.py [hunk 1]
```diff
@@ -1615,6 +1617,12 @@ def _handle_button_press(self, action):
             elif action == "nav_menu":
                 self._canvas.timeselection_dialog.open()
 
+        elif action.startswith("select_"):
+            # A selection action
+            if action == 'select_none':
```
Comment (almarklein) [inline timetagger/app/front.py:1622]:
```suggestion
            if action == "select_none":
```

**Comment by almarklein** [comment]
Thanks! I forgot about this PR. I fixed the formatting and merged.

In the future, you can run `black .` on the repo to fix formatting.

**Comment by Tagirijus** [comment]
Cool thanks for the info and thanks for merging. I recently found a bug, though. When pressing `Backspace` now the filtered tags do not get re-shown in the list. I have to investigate - just wanted to note this down. If I can find a solution I simply will probably come up with another PR. Thanks for merging and thanks for checking my code. (=

**Comment by almarklein** [comment]
I cannot reproduce the bug, and also don't see how that could be happening. For good measure I did #497 so `._tag_bars_dict` is also reset.

**Pull request merged**